hazelcast-sql/src/test/java/com/hazelcast/jet/sql/SqlClientPartitionAwareRoutingTest.java

```diff
@@ -0,0 +1,113 @@
+/*
```

Comment:
After a second thought, this might not be really necessary, but it would be good to have it if it is easy to do. Otherwise, feel free to ignore this comment